common/scripted_effects/wc_sacrifice_scripted_effects.txt

```diff
@@ -257,4 +257,18 @@ PREV_sacrifice_THIS_to_old_god_effect = {
 		}
 		PREV_society_sacrifice_THIS_effect = yes
 	}
+}
+
+sacrifice_all_prisoners_decision_effect = {
+	hidden_effect = {
+		any_courtier = {
+			limit = {
+				THIS_is_prisoner_that_can_be_sacrificed_trigger = yes
+				host = { character = ROOT }
+			}
+			caltulate_PREV_offmap_old_gods_currency_insanity_effect = yes
+			PREV = { add_offmap_old_gods_currency_insanity_after_calculation_effect = yes }
+			PREV_public_sacrifice_THIS_effect = yes
```

Comment:
The TH cult can sacriface characters, but doesn't get currency.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/76124603-82e6d600-6014-11ea-98b6-bbd1f62cd9ff.png)
![image](https://user-images.githubusercontent.com/46384992/76124393-fc31f900-6013-11ea-8dcc-ee234c345955.png)
![image](https://user-images.githubusercontent.com/46384992/76124401-0522ca80-6014-11ea-9791-49d910f84044.png)

</details>

Reply:
> The TH cult can sacriface characters

No, being in TH cult don't give you ability to sacrifice ***all*** prisoners.
Being openly OG does.

> but doesn't get currency

That is intended.
Since massive sacrifice is done openly, not secretly, like in society, so your character always gets piety instead.

Reply:
> No, being in TH cult don't give you ability to sacrifice _**all**_ prisoners.

Are there any obstacles you see?

Reply:
1 - If character is a secret cultist, this decision can increase visibility by... a lot. Yeah, player may expected it, but if it's not, then it can become big problem for them.
2 - TH don't really need mass sacrifice, since they get only 10 points for "non-special" characters.

But I think I can add society currency instead of piety if your character is both openly OG and in TH cult, whaddya think?

Reply:
> But I think I can add society currency instead of piety if your character is both openly OG and in TH cult, whaddya think?

@MemeWizard sry, missed your comment, I'll open a suggestion.

Reply:
Opened, https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/1309.